src/mscorlib/src/System/Text/DecoderBestFitFallback.cs

```diff
@@ -34,13 +31,7 @@ public override DecoderFallbackBuffer CreateFallbackBuffer()
         }
 
         // Maximum number of characters that this instance of this fallback could return
-        public override int MaxCharCount
-        {
-            get
-            {
-                return 1;
-            }
-        }
+        public override int MaxCharCount => 1;
```

Comment:
System.Text is far prettier in CoreRT; can they be swapped wholesale or is it a partial overlap?